config.yaml

```diff
@@ -6,3 +6,11 @@ options:
      Enable GRUB cgroup overrides cgroup_enable=memory swapaccount=1. WARNING
      changing this option will reboot the host - use with caution on production
      services
+  http_proxy:
+     type: string
+     default:
+     description: HTTP proxy to embed as an environment setting for Docker
```

Comment:
I find this wording a bit hard to understand.  Perhaps something like "URL to use for HTTP_PROXY to be used by Docker" instead?  Also, I'm unclear what this config option is for, why you would want to use it.  Does it relate in any way to the http_proxy setting that Juju supports?  Would it make sense to pass through the HTTP_PROXY from the charm's environment?